blocks/subheadline-block/features/subheadline/default.jsx

```diff
@@ -18,15 +15,9 @@ export const SubHeadlinePresentation = ({
 		valueToDisplay === DISPLAY_VALUES.DESCRIPTION
 			? content?.description?.basic
 			: content?.subheadlines?.basic;
-	return (
-		!!value && (
-			<PrimaryFont
-				as="h2"
-				className="h4-primary sub-headline"
-				dangerouslySetInnerHTML={{ __html: value }}
-			/>
-		)
-	);
+	return value ? (
+		<h2 className="b-subheadline" dangerouslySetInnerHTML={{ __html: value }} />
```

Comment:
This will always render as an h2 similar to headline only using h1, right? only the header is using heading/heading section based on my my reading https://arcpublishing.atlassian.net/jira/software/c/projects/TMEDIA/boards/875?modal=detail&selectedIssue=TMEDIA-712

Reply:
@matthewroach what do you think? 

Is subheadline always going to be an h2? or will it use the heading section/heading architecture? I was using the h2 always as the headline it’s commonly used with will always be an h1 and does not use the heading section component either

Reply:
Curious about @JackHowa comment above.  Only thing imo holding this ticket up

Reply:
Switching to HeadingSection > Heading would give the same output by default - as this block relies on global content it does have a limited use case on where it can be placed and ideally it would come after the Headline component. For consistency I think we should update it to use the HeadingSection > Heading set up